src/routes/LandingPageContent.ts

```diff
@@ -0,0 +1,94 @@
+import { ProjectEntry } from "../types";
+
+export const landingPageContent: ProjectEntry[] = [
+  {
+    name: "NucMorph dataset name which usually is a rather long name so may wrap to two lines",
+    description:
+      "Introductory explanatory text about the dataset(s) and anything a user should know before opening in app. This should ideally only be a couple of sentences.",
+    publicationLink: new URL("https://www.google.com"),
+    publicationName:
+      "This is the name of the associated publication that the user can click to open in a new tab (Publisher name, mm/dd/yyyy)",
+    loadLink:
+      "viewer?collection=https%3A%2F%2Fdev-aics-dtp-001.int.allencell.org%2Fdan-data%2Fcolorizer%2Fdata%2Fcollection.json&dataset=Mama+Bear&feature=Volume&color=matplotlib-cool",
```

Comment:
I'm curious about this load link. First it seems odd to me to include the viewer path if you always know that any dataset is going to get loaded at the viewer path. this also seems like it would be tedious to edit if you decided you wanted a different default color map, for example, and had a bunch of params in this link. I feel like this could be separated out into `defaultSettings` and `collectionUrl`. Is `dataset=Mama+Bear` just the dataset name, or does that get used to load some other data?

Reply:
Oh, breaking this up makes sense, thank you for the suggestion! I think I can reuse an existing datastructure I have that serializes the state to URL parameters.

Yes, the `dataset` URL param determines which dataset in the collection to load.

Reply:
Change has been made! See https://github.com/allen-cell-animated/nucmorph-colorizer/pull/260/files/a438f1b1f6223d4bd6eb2e8e5c5d43d55f34193d#diff-845b967d105e866efc166956c15fa7b2a2fab3dad00bdeb978f06a17c671da81.

I am a bit worried that this will make it harder for other people to make changes to the landing page content; while it's more maintainable for the developer side, it's a higher barrier of entry for everyone else. Any thoughts @toloudis ?